tests/utils/PolygonConnectorTest.cpp

```diff
@@ -61,16 +60,22 @@ class PolygonConnectorTest : public testing::Test
  */
 TEST_F(PolygonConnectorTest, getBridgeNestedSquares)
 {
-    std::vector<Polygon> to_connect({test_square_around});
+    std::vector<Polygon> to_connect({ test_square_around });
     std::optional<PolygonConnector::PolygonBridge<Polygon>> bridge = pc->getBridge(test_square, to_connect);
 
-    ASSERT_NE(bridge, std::nullopt) << "The two polygons are nested simply, so they are definitely positioned closely enough to bridge. They are also wide enough.";
+    ASSERT_NE(bridge, std::nullopt)
+      << "The two polygons are nested simply, so they are definitely positioned closely enough to bridge. They are also wide enough.";
 
-    EXPECT_EQ(vSize(bridge->a.from_point - bridge->a.to_point), 100) << "The polygons are 100 units spaced out concentrically, so this is the shortest possible bridge.";
+    EXPECT_EQ(vSize(bridge->a.from_point - bridge->a.to_point), 100)
+      << "The polygons are 100 units spaced out concentrically, so this is the shortest possible bridge.";
     EXPECT_EQ(vSize(bridge->b.from_point - bridge->b.to_point), 100) << "The second bridge should also be equally short in this case.";
-    EXPECT_EQ(LinearAlg2D::getDist2BetweenLineSegments(bridge->a.from_point, bridge->a.to_point, bridge->b.from_point, bridge->b.to_point), 100 * 100) << "The bridges should be spaced 1 line width (100 units) apart.";
-    EXPECT_LT(LinearAlg2D::pointIsLeftOfLine(bridge->b.from_point, bridge->a.from_point, bridge->a.to_point), 0) << "Connection B should be to the right of connection A.";
-    EXPECT_LT(LinearAlg2D::pointIsLeftOfLine(bridge->b.to_point, bridge->a.from_point, bridge->a.to_point), 0) << "Connection B should be to the right of connection A.";
+    EXPECT_EQ(LinearAlg2D::getDist2BetweenLineSegments(bridge->a.from_point, bridge->a.to_point, bridge->b.from_point, bridge->b.to_point),
+              100 * 100)
```

Comment:
I don't think that breaking this line into two pieces makes it any more readable (see also the copy below).